src/stories/HdInputFormatter.stories.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from '@storybook/vue';
+import { action } from '@storybook/addon-actions';
+import { HdInputFormatter } from 'homeday-blocks';
+import FormWrapper from 'homeday-blocks/src/storiesWrappers/FormWrapper';
+
+storiesOf('Components|Form/HdInputFormatter', module)
+  .addDecorator(FormWrapper)
+  .add('simple', () => ({
+    components: { HdInputFormatter },
+    template: `
+      <HdInputFormatter
+        v-model="value"
+        :formatter="formatter"
+        name="test"
+        label="Currency formatter"
+        type="number"
+      />
+    `,
+    data() {
+      return {
+        value: 123,
+      };
+    },
+    methods: {
+      formatter(value) {
+        if (value) {
+          return `$ ${value}`;
+        }
+
+        return '';
+      },
+    },
+    watch: {
+      value(value) {
+        action('input')(value);
+      },
+    },
+  }), { percy: { skip: true } });
```

Comment:
You're skipping it because it's based on HdInput?

Reply:
Yep, shall I add a comment? If so, could you make a `insert suggestion`? =]

Reply:
Here you go:
```suggestion
  }), {
    // Doesn't make sense to test the UI of this components because it's based on HdInput
    percy: { skip: true },
  });
```